wvd-templates/Create and provision WVD host pool/mainTemplate.json

```diff
@@ -91,6 +91,13 @@
             },
             "defaultValue": "Standard_D2s_v3"
         },
+        "enableAcceleratedNetworking": {
```

Comment:
Can we add a link for more information? Or mention that it only applies to Azure Gallery images of Windows Server?

Reply:
I can add the link for AN but as I explained in a previous comment it is supported from Templates, Azure CLI and Powershell, just the portal can't identify and disables from the regular VM deployment wizard.